index.php

```diff
@@ -7,37 +7,69 @@
  * @author Dominik Brader <support@findologic.com>
  */
 
-if(!defined('DOKU_INC')) {
-    define('DOKU_INC',realpath(dirname(__FILE__).'/../../../').'/');
-}
-// Get $conf from DokuWiki for constructor
-global $conf;
+require_once(__DIR__ . '/DokuwikiXMLExport.php');
```

Comment:
I won't require that for passing the review, but if you like you can wrap all the stuff (consts, methods) in a small class; classes are always good for maintainability and they don't cost nothin'

Reply:
Will implement this. It only makes sense.